scripts/exobsmon_plot.sh

```diff
@@ -24,31 +20,89 @@ if [[ ! -e ${chan_yaml} ]]; then
    exit 2
 fi
 
-#----------------------------------------
-# split $plot_yaml into sat/instr[/plot]
+#-----------------------------------------------------------
+# split $plot_yaml into sat/instr[/plot], minimization, obs
 #
 ${APRUN_PY} ${USHobsmon}/splitPlotYaml.py -i ${plot_yaml} -c ${chan_yaml}
 
-#exit 0
 
-cmdfile="OM_test_jobscript"
->$cmdfile
+#--------------------------------------------------------------
+# Submit OM_sat_plots job if split yields any sat_*.yaml files
+#
+if compgen -G "./sat_*.yaml" > /dev/null; then
+
+   jobname="OM_sat_plots"
+   logfile="${OM_LOGS}/${MODEL}/OM_sat_plot.log"
+   if [[ -e ${logfile} ]]; then rm ${logfile}; fi
+
+   cmdfile="OM_sat_jobscript"
+   >$cmdfile
+   ctr=0
+
+   for yaml in ./sat_*.yaml; do
```

Comment:
perhaps define a path as a var instead of `./` here and above?

Reply:
Change made here, above, and in 3 or 4 places below.  Good idea, thanks.